src/main/java/tc/oc/pgm/util/NameCacheUtil.java

```diff
@@ -13,26 +12,27 @@
 import java.net.HttpURLConnection;
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.List;
-import java.util.Map.Entry;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
 import java.util.UUID;
 import org.bukkit.Bukkit;
 import tc.oc.pgm.api.PGM;
 
 public class NameCacheUtil {
 
   private static final Gson GSON = new Gson();
+  private static final long ONE_WEEK_AGO = 7 * 24 * 60 * 60 * 1000;
```

Comment:
This can't be a constant as time moves on, it needs to be inside `readCacheFromDisk`, maybe keep this as `ONE_WEEK` and have the method use `long ONE_WEEK_AGO = System.currentTimeMillis() - ONE_WEEK`;